backend/src/main/scala/bloop/tracing/TraceProperties.scala

```diff
@@ -0,0 +1,79 @@
+package bloop.tracing
+
+import scala.util.Properties
+
+case class TraceProperties(
+    zipkinServerUrl: Option[String],
+    debug: Option[Boolean],
+    verbose: Option[Boolean],
+    localServiceName: Option[String],
```

Comment:
Should we call this `serviceName`? Not sure if it makes sense to distinguish between "local" service names and just normal service names.